internal/gcs/bridge.go

```diff
@@ -298,7 +303,11 @@ func (brdg *bridge) recvLoop() error {
 		brdg.log.WithFields(logrus.Fields{
 			"payload":    string(b),
 			"type":       typ.String(),
-			"message-id": id}).Debug("bridge receive")
+			"message-id": id}).Trace("bridge receive")
+		span.AddAttributes(
+			trace.StringAttribute("type", typ.String()),
+			trace.Int64Attribute("message-id", id))
```

Comment:
This will change the span properties for each loop iteration, since the span is created only once outside the `for {}` loop

Reply:
Moving this span to the readMessage() call above this funciton